From 77f6df5f118c2c599fbbfbc4263db07d88be2c5e Mon Sep 17 00:00:00 2001
From: agaraix <anath.bandux.garai@intel.com>
Date: Wed, 1 May 2019 20:56:51 +0530
Subject: [PATCH] PPA : Fix compile warning.

Include qos_hal_api.h for 'struct qos_hal_qos_stats' declaration.
---
 drivers/net/ethernet/lantiq/ppa/ppa_drv_wrapper.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/lantiq/ppa/ppa_drv_wrapper.c b/drivers/net/ethernet/lantiq/ppa/ppa_drv_wrapper.c
index f7ecd9afab86..cba1dec5bfba 100644
--- a/drivers/net/ethernet/lantiq/ppa/ppa_drv_wrapper.c
+++ b/drivers/net/ethernet/lantiq/ppa/ppa_drv_wrapper.c
@@ -44,6 +44,7 @@
 #include <net/ppa/ppa_api.h>
 #include <net/ppa/ppa_hal_api.h>
 #include <net/ppa/ppa_drv_wrapper.h>
+#include <net/ppa/qos_hal_api.h>
 
 #if IS_ENABLED(CONFIG_SOC_GRX500)
 #include <net/datapath_api.h>
